### DISCUSSION TITLE
How to install termux on android 13?

### DISCUSSION THREAD

**Discussion by AbcDyi** [termux/termux-app#3367] [Q&A] [answered]
问问...........来个大神

**Comment by 2096779623** [comment, marked as answer]
**Please use english!**
https://mirrors.tuna.tsinghua.edu.cn/fdroid/repo/com.termux_118.apk

**Reply by xinliushi** [reply]
thx bro!

**Comment by haviq** [comment]
How to fix termux unable to install boostrap in Android 13?

**Reply by agnostic-apollo** [reply]
Force stop the app and open again, otherwise post error.

**Comment by haviq** [comment]
Still the same, I've re-downloaded the universal version

Pada Rab, 3 Jan 2024 08.22, agnostic-apollo ***@***.***>
menulis:
